test/inspect.test.js

```diff
@@ -171,3 +171,20 @@ tap.test('versions inacurracy when composer is not installed', function (t) {
       t.end();
     }).catch(tap.threw);
 });
+
+tap.test('project name is not empty', function (t) {
+  var projFolder = './test/stubs/no_project_name';
+
+  return plugin.inspect(projFolder, 'composer.lock', options)
+    .then(function (result) {
+      t.test('make sure project name is no-name', function (t) {
+        t.deepEqual(
+          result.package.name,
+          'snyk-php-plugin'
```

Comment:
how does this pass? shouldn't it be 'no_project_name' ?

Reply:
the cwd is the directory where the process is ran from (we run tap from the main project dir)